tests/python_elif_multi_body/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,14 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`elif_multi_body.py 1`] = `
+if condition:
+    pass
+else:
+    assert something
+    another_statement()
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+if condition:
+    pass
+assert somethinganother_statement()
+
```

Comment:
Is this intentional?

Reply:
Oh wow no. I made it not crash, I didn't realize it was completely broken... Oops!